dbfit-java/build.gradle

```diff
@@ -83,7 +83,75 @@ subprojects {
         include('**/*Support*')
     }
 
-    task integrationTest(type: Test) {
+    task deploymentProperties {
+        doLast {
+            ext {
+                delimiterType = 'normal'
+                delimiter = ';'
+                setUpTestScriptPath = "${rootProject.projectDir}/test_vm/scripts/${project.name}/sql/"
+                setUpTestDbScript = setUpTestScriptPath + "create-databases-${project.name}.sql"
+                tearDownTestDbScript = setUpTestScriptPath + "drop-databases-${project.name}.sql"
+            }
+        }
+    }
+
+    // to be overriden when needed
+    ext.connectionPropertiesFiles = []
+
+    task connectionFileProperties {
+        doLast {
+            copyConnectionProperties '.'
+            ext {
+                userName = ''
+                passWord = ''
+                service = ''
+                database = ''
+            }
+            if (connectionPropertiesFiles.size() > 0) {
+                def props = new Properties()
+                file(connectionPropertiesFiles[0]).withInputStream { props.load(it) }
```

Comment:
We have that props loading from a file in several places. I'm wondering can we reuse the implementation.